apps/masterbots.ai/components/routes/browse/browse-list.tsx

```
@@ -100,22 +109,41 @@ export default function BrowseList() {
-          loading={loading}
-          loadMore={loadMore}
-          hasMore={count === PAGE_SIZE}
-          isLast={key === filteredThreads.length - 1}
-        />
-      ))}
+      {filteredThreads.length > 0 ? (
+        <>
+          {filteredThreads.map((thread: Thread, key) => (
+            <BrowseListItem
+              thread={thread}
+              key={key}
```

Biome

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)